2024Robot_Team3484/src/main/cpp/subsystems/ShooterSubsystem.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include "subsystems/ShooterSubsystem.h"
+
+#include <numbers>
+
+using namespace ShooterConstants;
+
+ShooterSubsystem::ShooterSubsystem() {
+
+    SH_Left_motor.RestoreFactoryDefaults();
+    SH_Right_motor.RestoreFactoryDefaults();
+
+    SH_Left_motor.SetInverted(MOTER_LEFT_INVERTED);
+    SH_Right_motor.SetInverted(!MOTER_LEFT_INVERTED);
+
+    SH_m_Left_pidController.SetP(P);
+    SH_m_Left_pidController.SetI(I);
+    SH_m_Left_pidController.SetD(D);
+    SH_m_Left_pidController.SetIZone(0);
+    SH_m_Left_pidController.SetFF(FF);
+    SH_m_Left_pidController.SetOutputRange(0, 1);
```

Comment:
I would suggest passing these PID values in through the constructor (you can consolidate the parameter set by using `SC_PIDConstants` found [here](https://github.com/FRC-Team3484/X24_RobotCode/blob/0ab6465eace9fa6ea8e356b1907560f294b84a82/2024Robot_Team3484/src/main/include/FRC3484_Lib/utils/SC_Datatypes.h#L63)